pr_check.sh

```
@@ -303,7 +356,17 @@ send_report() {
     fi
 
     MESSAGE="$MESSAGE $JOB_URL"
-    
+
+    # Check for open infra-deployments PRs and append to message
+    if [ -f "artifacts/infra-deployments-prs.json" ]; then
+        local pr_count=$(jq '. | length' artifacts/infra-deployments-prs.json 2>/dev/null || echo "0")
+        if [ "$pr_count" -gt 0 ]; then
+            MESSAGE="$MESSAGE\n\n📋 *Open infra-deployments PRs ($pr_count):*"
+            local pr_list=$(jq -r '.[] | "\n• <\(.url)|#\(.number)>: \(.title) (created \(.createdAt | split("T")[0]))"' artifacts/infra-deployments-prs.json 2>/dev/null)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)